nest/command_line.py

```diff
@@ -16,18 +16,20 @@
 from . import utils
 from . import helpers
 
+# use six for python2/python3 compatibility
+from six.moves import input
```

Comment:
Added to allow use of input in both Python2 and Python3 with it functioning correctly.